app/src/main/java/com/teamwss/websoso/ui/main/home/HomeViewModel.kt

```diff
@@ -68,19 +73,28 @@ class HomeViewModel @Inject constructor(
             runCatching {
                 feedRepository.fetchUserInterestFeeds()
             }.onSuccess { userInterestFeeds ->
+                val isInterestNovel =
+                    when (UserInterestFeedMessage.fromMessage(userInterestFeeds.message)) {
+                        UserInterestFeedMessage.NO_ASSOCIATED_FEEDS -> true
```

Comment:
c: 여기 UserInterestFeedMessage import 하면 좋을 것 같아요 👍

Reply:
UserInterestFeedMessage는 이미 임포트가 되어있습니다

Reply:
옵션 + 엔터 누르면 import 멤버스 어쩌구 ~ 로 임포트 해서 
```suggestion
                        NO_ASSOCIATED_FEEDS -> true
```
이렇게 쓸 수 있습니다

Reply:
아! 아래 enum class ㄷㄷ